### DISCUSSION TITLE
Do not get all metrics

### DISCUSSION THREAD

**Discussion by Tealk** [goauthentik/authentik#7156] [Q&A] [answered]
Hello all,

I wanted to build a Grafana dashboard together but get from authenik from prometheus only the following:

- scrape_duration_seconds{
- scrape_samples_post_metric_relabeling
- scrape_samples_scraped
- scrape_series_added
- up

what am i doing wrong?

**Comment by BeryJu** [comment]
Are you scraping port 9300 on the server container? See https://goauthentik.io/docs/installation/monitoring

**Reply by Tealk** [reply]
yes

```
  - job_name: "authentik"
    static_configs:
      - targets: ["192.168.100.34:9300"]
```

**Comment by Tealk** [comment, marked as answer]
the problem was too simple, in the docker-compose file the port was not shared.

docker-compose.yml
```
    ports:
      - "${COMPOSE_PORT_HTTP:-9000}:9000"
      - "${COMPOSE_PORT_HTTPS:-9443}:9443"
      - "9300:9300"
```
